Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -55,6 +51,58 @@ async function run(): Promise<void> {
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
 
+#if WIF
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
+        const feedIdNames = tl.getDelimitedInput("artifactsFeeds", ',');
+
+        if (entraWifServiceConnectionName) {
+            
+            tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName));
+            let token = await getFederatedWorkloadIdentityCredentials(entraWifServiceConnectionName);
+            
+            if (token) {
+
+                const centralElement = {
+                    id: 'central',
+                    username: 'CentralBuildWIF',
+                    password: token
+                }
+
+                settingsJson = util.addRepositoryEntryToSettingsJson(settingsJson, centralElement);
+                federatedFeedAuthSuccessCount++;
+
+                for (let feedName of feedIdNames) {
+                    const wifServerElement = {
+                        id: feedName,
+                        username: 'WIFbuild',
+                        password: token
+                    };
+    
+                    settingsJson = util.addRepositoryEntryToSettingsJson(settingsJson, wifServerElement);
+                    federatedFeedAuthSuccessCount++;
+                    console.log(tl.loc("Info_SuccessAddingFederatedFeedAuth", feedName));
+                }
+
+                tl.debug(tl.loc("Info_WritingToSettingsXml"));
+                await util.jsonToXmlConverter(userSettingsXmlPath, settingsJson);
+
+            }
+            else {
```

Comment:
I dont think we want this else statement anymore - wouldnt this block any other code flow if not set ?